pkg/manifestclient/list.go

```diff
@@ -241,7 +241,7 @@ func allPossibleListFileLocations(sourceFS fs.FS, requestInfo *apirequest.Reques
 
 		namespaces, err := allNamespacesWithData(sourceFS)
 		if err != nil {
```

Comment:
is this a not found error in your case?  If so, how about we ignore only that error.

Reply:
eh, it turns out that we already have a test that lists `cluster-scoped-resources` without the namespace directory.

The test works because `cluster-scoped-resources` are expected to be nested in a directory.

For example: `cluster-scoped-resources/operator.openshift.io/authentications/cluster.yaml`
NOT: `cluster-scoped-resources/operator.openshift.io/authentications.yaml`

I copy-pasted the directory structure from MOM, which apparently requires a fix. I will issue a separate PR for that.

Reply:
heh, okay, actually both are valid and can be found in a must-gather dir.